### DISCUSSION TITLE
Error Lens-style LSP diagnostics?

### DISCUSSION THREAD

**Discussion by flippette** [helix-editor/helix#6537] [Ideas] [answered]
Can we have LSP diagnostics inline like the Error Lens extension in VS Code?

This is what I'm talking about:

Helix:
![image](https://user-images.githubusercontent.com/43138244/229296289-c94632c5-a20d-48e2-a73d-67e8ff7d5f8a.png)

Error Lens:
![image](https://user-images.githubusercontent.com/43138244/229296342-9a5b3b8f-2363-4ba5-8f98-84c989f92450.png)

**Comment by David-Else** [comment, marked as answer]
https://github.com/helix-editor/helix/pull/6417
